index.html

```diff
@@ -0,0 +1,200 @@
+    <title>The Burning Circle Airshow 2021</title>
+  </head>
+  <body>
+    <div class="top-section">
+      <div class="bar-container">
+        <ul class="bars">
+          <li>
+            <i class="fab fa-facebook-f"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter"></i>
+          </li>
+          <li>
+            <h4>English</h4>
+          </li>
+          <li>
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <header>
+      <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"
+            ><img src="images/plane3.svg" alt="plane" class="plane"
+          /></a>
+          <div class="menu-bar nav" id="menu">
+            <i class="fas fa-bars fa-3x"></i>
+          </div>
+          <div class="cancel" id="close">
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+
+          <div class="company">
+            <a href="index.html">Burning Circle</a>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <section class="section-one">
+      <div class="section-one-container">
+        <h2 class="intro">
+          "Get the chance to meet Aviation Experts and get the chance to have a
+          look at different types of aircraft the world has to offer"
+        </h2>
+        <h1 class="heading">Burning Circle Flight Academy Airshow</h1>
+        <div class="paragraph1">
+          <p>
+            The Burining Circle Flight Academy airshow is one of Zambia's first
+            airshows to be hosted by a privately owned aviation institution. The
+            whole essence of this event is to connect aerospace industry
+            proffessionals across Africa and the world at Large.
+          </p>
+        </div>
+        <div class="date">
+          <h3>2021.11.14(SUN) ~ 18(THURS)</h3>
+          <p class="venue">@Kenneth Kaunda International Airport</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="section-two">
+      <div class="section-two-container">
+        <h2 class="heading2">Main Program</h2>
+        <hr />
+        <div class="program">
+          <div class="pilot-training stacks">
+            <img class="icons" src="images/program1.png" alt="" />
```

Comment:
- [ ] The `Home` page has many empty `alt` values for images (11), please update them with proper text.

Reply:
done